diff --git a/include/configs/ti_omap5_common.h b/include/configs/ti_omap5_common.h
index 6c813538825..52e3dea3bd5 100644
--- a/include/configs/ti_omap5_common.h
+++ b/include/configs/ti_omap5_common.h
@@ -155,8 +155,8 @@
 
 #define DEFAULT_COMMON_BOOT_TI_ARGS \
 	"console=" CONSOLEDEV ",115200n8\0" \
-	"fdtfile=undefined\0" \
-	"finduuid=part uuid mmc 0:2 uuid\0" \
+	"fdtfile=undefined\0" \	
+	"finduuid=if mmc dev 0; then part uuid mmc 0:2 uuid; elif mmc dev 1; then part uuid mmc 1:2 uuid; fi\0" \
 	"usbtty=cdc_acm\0" \
 	"vram=16M\0" \
 	AVB_VERIFY_CMD \
